src/OrchardCore.Modules/OrchardCore.ContentFields/Views/ContentPickerFieldSettings.Edit.cshtml

```diff
@@ -0,0 +1,60 @@
+@model OrchardCore.ContentFields.Settings.ContentPickerFieldSettings
+@using OrchardCore.ContentManagement;
+
+@inject IEnumerable<IContentPickerResultProvider> ResultProviders
+
+@{
+    var resultProviderItems = ResultProviders.Select(p => new SelectListItem
+    {
+        Text = p.Name,
+        Value = p.Name
+    });
+}
+
+<fieldset class="form-group">
+    <div class="form-check">
+        <label class="form-check-label">
+            <input asp-for="Required" type="checkbox" class="form-check-input" />
+            <span class="hint">@T["Check if the field is required."]</span>
+        </label>
+    </div>
+</fieldset>
+
+<fieldset class="form-group">
+    <div class="form-check">
+        <label class="form-check-label">
+            <input asp-for="Multiple" type="checkbox" class="form-check-input" />
+            <span class="hint">@T["Check to allow the user to select multiple content items."]</span>
+        </label>
+    </div>
+</fieldset>
+
+<fieldset class="form-group">
+    <label asp-for="DisplayedContentTypes">@T["Content Types"]</label>
+    <span class="hint">@T["The content types to display. Choose at least one."]</span>
+    @await Component.InvokeAsync("SelectContentTypes", new { selectedContentTypes = Model.DisplayedContentTypes, htmlName = Html.NameFor(m => m.DisplayedContentTypes) })
+</fieldset>
+
+<fieldset class="form-group">
+    <div class="row col-sm-6">
+        <label asp-for="Hint">@T["Hint"]</label>
+        <textarea asp-for="Hint" rows="2" class="form-control"></textarea>
+        <span class="hint">@T["The hint text to display for this field on the editor."]</span>
+    </div>
+</fieldset>
+
+<fieldset class="form-group">
+    <div class="row col-sm-6">
+        <label asp-for="SearchResultProvider">@T["Search Result Provider"]</label>
+        <select asp-for="SearchResultProvider" asp-items="resultProviderItems" class="form-control"></select>
+        <span class="hint">@T["The search result provider to use when selecting content items."]</span>
+    </div>
+</fieldset>
+
+<fieldset class="form-group">
+    <div class="row col-sm-6">
+        <label asp-for="SearchIndex">@T["Search Index"]</label>
+        <input asp-for="SearchIndex" type="text" class="form-control" />
+        <span class="hint">@T["The search index to use when using the Lucene provider."]</span>
```

Comment:
I was wondering how you were displaying this, unfortunately :/

Reply:
This should be a Lucene provider setting, like we have a Search setting. You can even integrate it in the same settings page by targetting the same group as the Search settings.